File: app/src/main/java/com/example/mealrecipeapp/MealRecipeApp.java
package com.example.mealrecipeapp;

import android.app.Application;

import com.example.mealrecipeapp.di.AppContainer;

public class MealRecipeApp extends Application {
    public AppContainer appContainer;

    @Override
    public void onCreate() {
        super.onCreate();
        appContainer = new AppContainer(this);
    }
}

File: app/src/main/java/com/example/mealrecipeapp/MealRecipeApp.kt
package com.example.mealrecipeapp

import android.app.Application
import com.example.mealrecipeapp.di.AppContainer

class MealRecipeApp : Application() {
    lateinit var appContainer: AppContainer
    override fun onCreate() {
        super.onCreate()
        appContainer = AppContainer(this)
    }
}
